test/parallel_api/algorithm/alg.nonmodifying/find_end.pass.cpp

```diff
@@ -90,8 +90,8 @@ test(const ::std::size_t bits)
 
     const ::std::size_t max_n1 = 1000;
     const ::std::size_t max_n2 = (max_n1 * 10) / 8;
-    Sequence<T> in(max_n1, [&](::std::size_t) { return T(2 * HashBits(max_n1, bits - 1) ^ 1); });
-    Sequence<T> sub(max_n2, [&](::std::size_t) { return T(2 * HashBits(max_n1, bits - 1)); });
+    Sequence<T> in(max_n1, [bits](::std::size_t) { return T(2 * HashBits(max_n1, bits - 1) ^ 1); });
+    Sequence<T> sub(max_n2, [bits](::std::size_t) { return T(2 * HashBits(max_n1, bits - 1)); });
```

Comment:
Why isn't an explicit or implicit capture of `max_n1` required?  Even though it is const, doesn't it still need to be captured in some way for use in the lambda?

Reply:
No, in this case it's not required. Otherwise we should have compile error.
In previous code we had the next warning here:
```
Warning G3C3A837B lambda capture 'max_n1' is not required to be captured for this use [-Wunused-lambda-capture]
```

Reply:
...but in C++20 only...

Reply:
Aha, I learned something new:  
https://en.cppreference.com/w/cpp/language/lambda

"A lambda expression can read the value of a variable without capturing it if the variable

has const non-volatile integral or enumeration type and has been initialized with a [constant expression](https://en.cppreference.com/w/cpp/language/constant_expression#Constant_expression), or
is constexpr and has no mutable members."

Reply:
To avoid compile error in C++17 I rolled back changes in this place.